resources/views/dashboard/controllers/files.blade.php

```diff
@@ -98,6 +122,25 @@
                 <tbody>
                     @if($vstars->count() > 0)
                         @foreach($vstars as $f)
+                            @if($f->row_separator == 1)
```

Comment:
We should be able to just use `@if($f->row_separator)` for these. 0 Should evaluate to false and anything greater than 0 is true. Laravel implements the boolean data type with a bit in MySQL (0 or 1)

Reply:
Done. Updated both views.